javagen/src/main/java/com/azure/autorest/model/clientmodel/ConvenienceExample.java

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.autorest.model.clientmodel;
+
+public class ConvenienceExample {
```

Comment:
Placeholder for `ConvenienceExample` for now. Will see if more parameters are needed.

Reply:
Maybe just MethodExample or ClientMethodExample?

We say protocol method example just because it is special. Convenience method is basically just client method in vanilla.

Reply:
Sounds good.